src/security/pci-guidance.md

```diff
@@ -0,0 +1,151 @@
+# PCI DSS Responsibility Guidance
+
+Platform.sh has many PCI certified customers using our services. Below we will attempt to clarify the PCI compliance responsibilities for Platform.sh and our customers.
```

Comment:
I heartily dislike "below we will" type language.  Recommended alternative:

"Some aspects of PCI compliance are the responsibility of the host, described below.  Others are the responsibility of the application developer."

Reply:
(That's just for the second sentence.  The first sentence is fine, but frankly extraneous.)